src/Copy/BrandCopy.elm

```diff
@@ -188,9 +189,17 @@ brandCopy key =
             CopyText "How secure is your data?"
 
         PrivacySectionSevenP ->
-            CopyText "We have procedures in place to protect your data. These include password protection on all Hospice Hope computers and devices; password protected telephones for Hospice Hope, locked, secure cabinets for all paper documents with limited keyholders. Trustees, staff and key volunteers are all informed of their responsibility relating to data security and all staff, trustees and volunteers sign a confidentiality statement before they begin to work or volunteer for Hospice Hope. Hospice Hope only shares data with approved agencies e.g. HMRC re Gift Aid or Social Services in case of any safeguarding incident. However, if you think your data has been given to another organisation without authorisation, please contact our Administrator on 07935 800 658 or email on administrator@hospicehope.org.uk."
+            CopySection
+                CopyText
```

Comment:
This doesn't look quite right.. should it be a list of a couple CopyTexts?

Reply:
Oops. Thanks.